### ISSUE TITLE
Gluon nn BatchNorm layer cannot ignore the beta

### ISSUE THREAD

**Issue by Will0622** [apache/incubator-mxnet#10401]
In mxnet symbol API, we cannot fix the beta to be 0. So, I tried to use the Gluon nn BatchNorm layer which allows setting center=False to ignore the beta term. However, When I use 
 bn = mx.nn.BatchNorm(scale=False, center=False)
I checked the mean of the output from this BatchNorm during training. I found that the mean is changing and also is significantly larger than 0 (e.g., it could be 0.7). Then, I looked at the implementation of the nn BatchNorm layer. It sets grad_req = 'null' and differentiable = False for beta when center = False. I have no clue why this is not working, or I got it wrong.

**Comment by szha** [comment]
Hi @Will0622. I wasn't able to reproduce this issue with simple code samples. Could you provide a snippet for reproducing the issue?

**Comment by szha** [comment]
@Will0622 closing for lack of activity. If this is still relevant, whenever you're ready, you can provide the information here and ping me to reopen the issue. Thanks.
